Commands/InteractionServiceHandler.cs

```diff
@@ -66,7 +67,7 @@ public static async Task OnInteractionCreatedAsync(SocketInteraction interaction
 		// The service might not have been initialized yet
 		if (_interactionService == null)
 		{
-			await Program.LogAsync("InteractionServiceManager.OnInteractionCreatedAsync()", "InteractionService not initialized yet", LogSeverity.Error);
+			Log.Error("InteractionService not initialized yet, InteractionServiceManager.OnInteractionCreatedAsync()");
```

Comment:
Same as in `RegisterModulesAsync()`

```suggestion
			Log.Error("Error handling interaction: InteractionService not initialized.");
```